wp-content/themes/missilethreat/inc/template-functions.php

```diff
@@ -341,20 +341,20 @@ function missilethreat_custom_sort_posts( $query ) {
  * @param  array $args  Menu.
  * @return string        Modified menu.
  */
-function missiledefense_add_search_box( $items, $args ) {
-    if($args->theme_location == 'primary') {
-        $search = '<li class="search">';
-        $search .= '<form method="get" id="searchform" action="/"><div class="input-group">';
-        $search .= '<label class="screen-reader-text" for="navSearchInput">Search for:</label>';
-        $search .= '<input type="text" class="form-control" name="s" id="navSearchInput" placeholder="Search" />';
-        $search .= '<label for="navSearchInput" id="navSearchLabel"><i class="fa fa-search" aria-hidden="true"></i></label>';
-        $search .= '</div></form>';
-        $search .= '</li>';
-        return $items.$search;
-    }
-    return $items;
-}
-add_filter( 'wp_nav_menu_items','missiledefense_add_search_box', 10, 2 );
+// function missiledefense_add_search_box( $items, $args ) {
```

Comment:
If you're not using this code anymore, remove it instead of commenting it out. We can always look it up in the git history if we need it again.